packages/dds/tree/src/test/simple-tree/schemaTypes.spec.ts

```diff
@@ -493,8 +493,15 @@ describe("schemaTypes", () => {
 			const schemaFactory = new SchemaFactory("test");
 
 			let Bar: TreeNodeSchema;
+
+			// eslint-disable-next-line no-constant-condition
```

Comment:
Could you accomplish the same thing via `const Bar: TreeNodeSchema = undefined as unknown as TreeNodeSchema;`?

Reply:
yes, but I like that the example as is actually is a case of use before assignment (which is why we bothered to add special undefined error cases) rather than just unsafe type casts (which we can't protect users from in general)